.env.example

```diff
@@ -7,7 +7,7 @@ OPERATOR_ADDRESS=tz1...
 
 # Teztnet network used (see https://teztnets.com/)
 NETWORK=ghostnet
-JSTZ_TAG=v0.1.0-...
+JSTZ_TAG=jstz-ghostnet
```

Comment:
Not to be added to the example env. `jstz-ghostnet` is just a tag you used locally.